### ISSUE TITLE
Restart causes failures on working cluster(s)

### ISSUE THREAD

**Issue by chuckbutler** [chuckbutler/docker-charm#23]
During installation of Kubernetes bundle - there is a docker container being pulled. During the flannel-docker relationship cycle - the docker daemon is actively pulling an image, and running cadvisor.

This is troublesome and causes races when the docker host is configuring itself.

**Comment by whitmo** [comment]
I wonder if setting the restart flag would effect this.  Also, does docker respond differently depending on the termination signal?

**Comment by chuckbutler** [comment]
You're right on this one @whitmo. The --restart flag seems to be the key to keeping the containers running consistently. I'll update the docs around known behavior, and close this out as there's very little to do here.
